examples/llm/configs/mutinode_disagg_r1.yaml

```diff
@@ -0,0 +1,46 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Just curious - have you tried on GB200 / Linux ARM machine?

Reply:
Haven't tried that